src/Components/reviewContainer.jsx

```diff
@@ -1,27 +1,42 @@
 import { useState, useEffect } from "react";
 import { listOfReviews } from "../utils/api";
 import { Link } from "react-router-dom";
+import { useLocation} from "react-router-dom";
+
+
+
 
 export default function ReviewContainer() {
   const [list, setList] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
-
+  const {search} = useLocation()
+  const [searched,setSearched]=useState()
+  const params = search.replace("?","")
+  
   useEffect(() => {
-    listOfReviews().then((response) => {
+    listOfReviews(params).then((response) => {
```

Comment:
See my comment in the `api.js` file about using proper axios params, it will help simplify this logic and you could use the `useParams()` hook